### ISSUE TITLE
Handling INSERT Intent on m5-rc14

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dragonlinux/connectbot#2]
```
I have the latest svn compiled and running on my emulator, but its not able
to launch the "Add Host" intent.

"No Activity found to handle Intent" is the error message.

However, the AndroidManifest.xml shows the intent-filter is correctly
defined.  The project is following the NotePad design closely, and I can't
see where the problem is.  Any thoughts?
```

Original issue reported on code.google.com by `Jeffrey.Sharkey` on 18 Apr 2008 at 6:23

**Comment by GoogleCodeExporter** [comment]
```
Found that some of the intents in AndroidManifest.xml weren't fully defined.  
Should
be changed similar to the following:

<data android:mimeType="vnd.android.cursor.item/vnd.theb.host" />

This means creating a new host works now on m5-rc14.

Another issue: the actual click-to-connect is broken.  The getSelectedItemId() 
in
onListItemClick is returning very odd values.  If you force it to index "1" you 
can
then successfully connect.

Another issue: you can't directly compile the SVN code in m5-rc14 because of 
issues
with Dalvik compiler.  I found a workaround for de.mud.terminal code, which I 
posted
in the official SDK issue:

http://code.google.com/p/android/issues/detail?id=246

Another issue: the key-exchange process seems to take forever in some cases, 
with the
emulator at full CPU load.  Sometimes the SSH can timeout before this process
finishes.  This might be an upstream issue in the Trilead SSH2 libraries used.

So with all of these changes, I've hacked an SSH connection together to work on 
the
m5-rc14 emulator.  :)
```

Original comment by `Jeffrey.Sharkey` on 19 Apr 2008 at 4:44

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
* r42 fixes the selected item ID
* r44 fixes the Add Host problem
```

Original comment by `kenny@the-b.org` on 27 Aug 2008 at 4:40
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Also, just a follow-up that the other issues (the Davlik compiler errors and 
SSH key
exchange time) have now been fixed by the 0.9 SDK.
```

Original comment by `Jeffrey.Sharkey` on 27 Aug 2008 at 5:51
